reco_cmsdriver2011.py

```diff
@@ -0,0 +1,86 @@
+# Auto generated configuration file
```

Comment:
git commit --author='John Doe <john.doe@cern.ch>'`

Reply:
For this I have used the actual command:
```console
cmsDriver.py reco -s RAW2DIGI,L1Reco,RECO,USER:EventFilter/HcalRawToDigi/hcallaserhbhehffilter2012_cff.hcallLaser2012Filter --data --conditions FT_R_53_LV5::All --eventcontent AOD --customise Configuration/DataProcessing/RecoTLR.customisePrompt --no_exec --python reco_cmsdriver2011.py
```
So it's hard to judge who actually came up with this, but from the commit history it's @diyaselis